### DISCUSSION TITLE
Publish client & schema npm packages for rc4

### DISCUSSION THREAD

**Discussion by mirceahasegan** [CardanoSolutions/ogmios#344] [Q & A]
Hello @KtorZ ,
Could you publish the latest ogmios/client & ogmios/schema packages, that are related to the ogmios 6.0rc4 release notes?

> [The integration with cardano-ledger-conway==1.9.0.0 also causes a few (breaking) changes to the existing interface](https://github.com/CardanoSolutions/ogmios/releases/tag/v6.0.0-rc4)

Thank you!

**Comment by KtorZ** [comment]
Whoops. This should have been done by the CI pipeline but somehow skipped because it's a pre-release. I've just done it manually 👍
